assets/js/player.js

```diff
@@ -199,6 +201,32 @@ if (video_data.premiere_timestamp && Math.round(new Date() / 1000) < video_data.
     player.getChild('bigPlayButton').hide();
 }
 
+if (video_data.params.save_player_pos) {
+    const url = new URL(location);
+    const hasTimeParam = url.searchParams.has("t");
+    const remeberedTime = get_video_time();
+    let lastUpdated = 0;
+
+    if(!hasTimeParam) {
+        set_seconds_after_start(remeberedTime);
+    }
+
+    const updateTime = () => {
+        const raw = player.currentTime();
+        const time = Math.floor(raw);
+
+        if(lastUpdated !== time) {
+            save_video_time(time);
+            lastUpdated = time;
+        }
```

Comment:
Btw, I recall that we discussed that somewhere, but can't find it: Could it be better to store the current time by increment of 5 seconds?